exercises/stack_01.js

```diff
@@ -0,0 +1,82 @@
+window.Collections = {};
+
+(function(module) {
+    function Stack(arr) {
+        /* constructor */
+        this.arr = arr;
+    };
+
+    /* creating a forEach method on Stack */
+    Stack.prototype.forEach = function(callback) {
+        for (var i = 0; i < this.arr.length; i++) {
+            callback(this.arr[i], i, this.arr);
+        }
+    };
+
+    Stack.prototype.push = function(elem){
+        // this.arr.push(elem);
+        this.arr[this.arr.length] = elem;
```

Comment:
The commented out version works fine, it turns out!

Reply:
Yep, left both since I didn't know how verbose it needed to be.